docs/releases/2023-december.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: December 2023
+parent: "Release notes"
+nav_order: 5
+---
+
+## Lance support
+
+**Available in**: [GlareDB@v0.7.1], [GlareDB Cloud]
+GlareDB now offers support for Lance.
+
+Lance is a Modern columnar data format for ML and LLMs. Read more about Lance [here](https://lancedb.github.io/lance/).
```

Comment:
**suggestion(optional)**: I'd have made `Lance` a link `[Lance]`, defined at bottom of page. Up to you, I think this also works well.